src/js/themes/base.js

```diff
@@ -517,17 +517,18 @@ export const generate = (baseSpacing = 24, scale = 6) => {
         side: 'bottom',
       },
       content: {
-        pad: {
-          horizontal: 'small',
-          bottom: 'small',
-        },
+        pad: 'small',
+      },
+      disabled: {
+        // background: undefined,
       },
       error: {
         color: {
           dark: 'status-critical',
           light: 'status-critical',
         },
         margin: { vertical: 'xsmall', horizontal: 'small' },
+        // background: undefined,
```

Comment:
Should be added in a similar way to https://github.com/grommet/grommet/blob/c4f9d42dc862bddde612172237a57aa7003c648a/src/js/themes/base.d.ts#L478

Reply:
yep